sbom/README.md

```diff
@@ -0,0 +1,13 @@
+# `sbom` Extension for Phylum
+This extension will take a provided lock file, submit it to Phylum for analysis and return the job data in structured SBOM format.
+
+## Installation and Basic Usage
+Clone the repository and install the extension via the Phylum CLI.
+
+```console
+git clone https://github.com/phylum-dev/community-extensions
+phylum extension install community-extensions/sbom/
+```
+
+## Running
+From a directory containing a lock file, run `phylum sbom <lockfile>`.
```

Comment:
This extension calls `phylum analyze`, which requires a phylum project. Just having a lock file is not a sufficient precondition:
<img width="791" alt="image" src="https://user-images.githubusercontent.com/1714777/193344163-8e8b649d-a531-479a-9b12-7f6c1ed933ee.png">

Reply:
Added a handler for this to give better instruction on failure.